airbyte-cdk/java/airbyte-cdk/typing-deduping/src/testFixtures/java/io/airbyte/integrations/base/destination/typing_deduping/RecordDiffer.java

```diff
@@ -432,10 +433,28 @@ private static LocalDate asDate(final JsonNode node) {
     }
   }
 
+  private static class Field implements Comparable<Field> {
+    String stringValue;
```

Comment:
Nit:  can these fields be private?  Does the wrapping class need access to them?